### ISSUE TITLE
VideoThumbnails are not generated

### ISSUE THREAD

**Issue by razyalov** [WWBN/AVideo#2445]
I have purchased the VideoThumbnails plugin and installed and enabled it.
For some reason, the thumbnails always show as a brown box instead of the actual thumbnail.
![Screen Shot 2019-12-27 at 2 12 45 AM](https://user-images.githubusercontent.com/3484169/71511138-827cd280-284e-11ea-914e-10fa11ca22c0.png)

**Comment by DanielnetoDotCom** [comment]
Looks like is something in your encoder

do you have the encoder properly working?

1 - try this, delete all thumbs ...

![image](https://user-images.githubusercontent.com/18009293/71518281-df00e180-2890-11ea-9077-33887f936aed.png)

2 - then delete your encoder log file and streamer log file.
3 - try to access a video (a thumbs request will be trigger)

check your encoder and streamer log file and share it here

**Comment by razyalov** [comment]
After following these steps above:
Streamer `avideo.log` (I masked some info with `xxxxxxx`):
```
[27-Dec-2019 17:05:22 UTC] createStprits: creating https://xxxxxxxxxx.us-west-2.amazonaws.com/_YPTuniqid_xxxxxxxxxb9.8xxxxx_Low.mp4
[27-Dec-2019 17:05:22 UTC] createStprits: https://xxxxxxxxx/video/encoder/getSpiritsFromVideo/aHR0XXXXXXXXXXXXXXFpZF81ZTA0MjdjZDBmODZiOS44OTY4MjY3N19Mb3cubXA0/150/100/3%3A42%3A27
```

Encoder `avideo.log` (again I replaced some values with `xxxxx`):
```
[23-Dec-2019 19:47:53 UTC] getSpritsFromVideo: php '/var/www/html/xxxxxxxxx/video/encoder/objects/createSpiritsFromVideo.php' 'https://xxxxxxxxx.s3.us-west-2.amazonaws.com/_YPTuniqid_5xxxxxxxx.xxxxxxxx7_Low.mp4' '7.27' '150' '84' '/var/www/html/xxxxxxxxx/video/encoder/videos/sprit_a3c3124d0de9b791ba5f23167691c1e9.jpg' '100' 'a3c3124d0de9b791ba5f23167691c1e9'
[23-Dec-2019 19:47:53 UTC] CreateSpirits:  creating directory /var/www/html/xxxxxx/video/encoder/videos/thumbs_a3c3124d0de9b791ba5f23167691c1e9/
[23-Dec-2019 19:47:53 UTC] CreateSpirits: ffmpeg -i "https://xxxxxxxxxxx.s3.us-west-2.amazonaws.com/_YPTuniqid_xxxxxxxxf857.xxxxxxx7_Low.mp4" -vf fps=1/7.27 -s 150x84 /var/www/html/xxxxxxxxx/video/encoder/videos/thumbs_a3c3124d0de9b791ba5f23167691c1e9/out%03d.png
[23-Dec-2019 19:47:53 UTC] CreateSpirits:  removing directory /var/www/html/xxxxxxxx/video/encoder/videos/thumbs_a3c3124d0de9b791ba5f23167691c1e9/
```

**Comment by akhilleusuggo** [comment]
@razyalov I see no errors there . I had the same problem . Try restarting apache2 and clearing cache .
Also open htop or other gui to see if the encoder is working and has finished .
Can you share the video link here ?

**Comment by razyalov** [comment]
I think I just found the issue. I tried troubleshooting by running the command manually and found that my `ffmpeg` is having trouble resolving dns names (it is a static version built for `x64` servers, as I run on `centos` which doesn't have `ffmpeg` in its package manager by default). I'm currently building ffmpeg from source to resolve this issue. Will report back.
Would be nice if `ffmpeg` errors were logged in the encoder log file.
